Sources/RxAlamofire.swift

```diff
@@ -859,35 +859,35 @@ extension Reactive where Base: DataRequest {
         return Observable.create { observer in
             self.base.downloadProgress { (progress) in
                 let rxProgress = RxProgress(bytesWritten: progress.completedUnitCount,
-                                            totalBytesWritten: progress.totalUnitCount - progress.completedUnitCount,
-                                            totalBytesExpectedToWrite: progress.totalUnitCount)
+                                            bytesRemaining: progress.totalUnitCount - progress.completedUnitCount,
+                                            totalBytes: progress.totalUnitCount)
                 observer.onNext(rxProgress)
-                if rxProgress.totalBytesWritten >= rxProgress.totalBytesExpectedToWrite {
+                if rxProgress.bytesWritten >= rxProgress.totalBytes {
                     observer.onCompleted()
                 }
             }
             return Disposables.create()
             }
             // warm up a bit :)
-            .startWith(RxProgress(bytesWritten: 0, totalBytesWritten: 0, totalBytesExpectedToWrite: 0))
+            .startWith(RxProgress(bytesWritten: 0, bytesRemaining: 0, totalBytes: 0))
     }
 }
 
 // MARK: RxProgress
 public struct RxProgress {
-    let bytesWritten: Int64
-    let totalBytesWritten: Int64
-    let totalBytesExpectedToWrite: Int64
+    public let bytesWritten: Int64
+    public let bytesRemaining: Int64
```

Comment:
It seems to me we can remove bytes remaining because it can be calculated from `totalBytes` and `bytesWritten`.

Maybe make `bytesRemaining` a calculated property in extension of `RxProgres`?